src/util/dict.c

```diff
@@ -302,6 +303,8 @@ long long timeInMilliseconds(void) {
 
 /* Rehash for an amount of time between ms milliseconds and ms+1 milliseconds */
 int dictRehashMilliseconds(dict *d, int ms) {
+    if (__atomic_load_n(&d->pauserehash, __ATOMIC_RELAXED) > 0) return 0;
```

Comment:
The function is used by a benchmark only and does need protection.

Reply:
dictPauseRehashing is a new API that exposed by the dictionary unit. It is better to support all over dictionary API even if not used...